### DISCUSSION TITLE
Can't I draw on the same layer with the brush tool after I paint with fill tool?

### DISCUSSION THREAD

**Discussion by hahahoho11** [keiwando/toonsquid#698] [Q&A] [answered]
Can't I draw on the same layer with the brush tool after I paint with fill tool?

I'm drawing with a brush tool on the fill tooled area by adding layers. Is this correct?

Can't I do it in the same layer?

**Comment by Cat-Butler** [comment, marked as answer]
You can only draw on that same layer if your brush type matches your layer type e.g. pixel brush on a pixel layer, vector brush on a vector brush layer. See the layer icons in the layers panel, and the presence/lack of a vector brush icon in the brush preview in the brush list.

Otherwise you'll get a new layer being automatically created above your selected layer.

**Reply by hahahoho11** [reply]
thank you for your reply!!
